config/initializers/omniauth.rb

```diff
@@ -0,0 +1,24 @@
+OmniAuth.config.logger = Rails.logger
+
+dfe_sign_in_issuer_uri    = URI(ENV.fetch("DFE_SIGN_IN_ISSUER"))
```

Comment:
I guess calling `fetch` here means the app will fall over in environments where these variables are not defined, e.g. the current temporary prod environment on Heroku. Just noting that here as it means we'll have to add the variables to the prod environment before this code is promoted there.